packages/cashier/src/pages/p2p-cashier/p2p-cashier.jsx

```diff
@@ -20,7 +21,9 @@ const P2PCashier = ({
     local_currency_config,
     location,
     loginid,
+    Notifications,
```

Comment:
Can we rename `Notifications` to stick to `snake_case` naming convention?

Reply:
It is used as a component so it's named that way.